sites/docs/src/content/docs/usage/configuration/system_requirements.md

```diff
@@ -0,0 +1,121 @@
+
+:::warning
+The `--max_<resource>` parameters do not represent the total sum of resource usage of the pipeline at a given time - only a single pipeline job!
+:::
+
+### Tuning resources
+
+The base config of nf-core pipelines defines the default resources allocated to each different step in the workflow (e.g., in a [`base.config`](https://github.com/nf-core/rnaseq/blob/master/conf/base.config) file).
+
+These values are deliberately generous due to the wide variety of workloads done by different users.
+As a result, you may find that the jobs are given more resources than they need and your system is not used efficiently.
+At the other end of the scale, you may want to increase the resources given to a specific task to make it run faster.
+You may wish to increase resources if you get a pipeline reporting a step failing with an `Command exit status`, such as `137`.
+
+Where possible, tools are tuned to make use of the resources available, for example with a tool parameter (e.g., `-p ${task.cpus}`, where `${task.cpus}` is dynamically set according to what has been specified in the pipeline configuration files). However, this is not possible with all tools, in which case required resources are estimated for an average user.
+
+Workflow resources can modified through [shared configuration profiles](#shared-configuration-files) or [custom configuration files](#custom-parameter-and-configuration-files) to match your requirements.
+
+Most process resources are specified using process labels by default. For example:
+
+```groovy
+process {
+  withLabel:process_low {
+    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
+    memory = { check_max( 14.GB * task.attempt, 'memory' ) }
+    time = { check_max( 6.h * task.attempt, 'time' ) }
+  }
+  withLabel:process_medium {
+    cpus = { check_max( 6 * task.attempt, 'cpus' ) }
+    memory = { check_max( 42.GB * task.attempt, 'memory' ) }
+    time = { check_max( 8.h * task.attempt, 'time' ) }
+  }
+  withLabel:process_high {
+    cpus = { check_max( 12 * task.attempt, 'cpus' ) }
+    memory = { check_max( 84.GB * task.attempt, 'memory' ) }
+    time = { check_max( 10.h * task.attempt, 'time' ) }
+  }
+}
+```
+
+The [`check_max()`](https://github.com/nf-core/tools/blob/99961bedab1518f592668727a4d692c4ddf3c336/nf_core/pipeline-template/nextflow.config#L206-L237) function applies the thresholds set in `--max_cpus`, `--max_memory` and `--max_time`. The `* task.attempt` means that these values are doubled and automatically retried after failing with an exit code that corresponds to a lack of resources.
```

Comment:
```suggestion
A default [`check_max()`](https://github.com/nf-core/tools/blob/99961bedab1518f592668727a4d692c4ddf3c336/nf_core/pipeline-template/nextflow.config#L206-L237) function will apply the thresholds set in `--max_cpus`, `--max_memory` and `--max_time` (or `params.max_cpus` etc. in  a config file).
The `* task.attempt` means that these values are doubled and automatically retried after failing with an exit code that corresponds to a lack of resources.
```

I sort of wodner if describing what the function does (E.g. with task.attempt etc) is necessary here...